betago/model.py

```diff
@@ -57,7 +65,7 @@ def next_move():
             json_result = jsonify(**result)
             return json_result
 
-        self.app.run(host='0.0.0.0', debug=True, threaded=True, use_reloader=False)
+        self.app.run(host='0.0.0.0', port=int("8081"), debug=True, threaded=True, use_reloader=False)
```

Comment:
Using command line flags will require piping the args through each script which is not something I have time to build and test.